examples/db/instance/main.tf

```diff
@@ -5,12 +5,13 @@ resource "vkcs_db_instance" "basic" {
   flavor_id         = data.vkcs_compute_flavor.basic.id
   datastore {
     type    = "mysql"
-    version = "5.7"
+    version = "8.0"
   }
+  configuration_id = vkcs_db_config_group.db-config-group.id
```

Comment:
lets place it before capabilities. and name it mysql

Reply:
i named it mysql-80 ( as i checked if config is verA and in instance datastore has verB there is an error)